### ISSUE TITLE
Facets as a Data Studio community report

### ISSUE THREAD

**Issue by avilella** [PAIR-code/facets#235]
Hi, has anyone thought of integrating Facets as a Google Data Studio community report? I see there are a bunch of cool reports added by third-party community members, such as violin plots, etc. but Facets is not in the list (or I couldn't find it).

**Comment by jameswex** [comment]
I wasn't aware of Google Data Studio third-party community reports visualizations. I think it would be possible to add Facets Dive and Facets Overview with a bit of work on bundling them up into the appropriate JS bundles and writing some adapter code between Data Studio and Facets.

**Comment by avilella** [comment]
Yes, as far as I can tell this is the way people can contribute to the visualizations. I have to say that my JS skills are next to zero, so I won't be able to contribute to it, so I'll cross my fingers that there is a willingness for this to be implemented in the Facets developers community.

**Comment by avilella** [comment]
Not sure if there is a micropayments method to encourage tickets in this repo @jameswex . Anything that can be done? Also, I believe there is now an open call for orgs to the Google Summer of Code 2021: "All organizations wishing to be a part of GSoC 2021 must complete their application by February 19, 2021 19:00 (Greenwich Mean Time).". https://summerofcode.withgoogle.com/how-it-works/. Happy to co-mentor from the point of view of the user of the feature.
